cl-gopher.lisp

```diff
@@ -478,16 +478,23 @@
             (if (and (>= (length uri) 9) (equal "gopher://" (subseq uri 0 9)))
                 (quri:uri uri)
                 (quri:uri (format nil "gopher://~a" uri))))
-           (path (quri:uri-path uri))
+           (path (quri:url-decode (quri:uri-path uri)))
            (item-type (compute-item-type uri path))
            (selector (compute-selector uri path))
+           (tab-split-selector (uiop:split-string selector :separator '(#\Tab)))
+           (selector (first tab-split-selector))
+           (terms (second tab-split-selector))
+           (gopher+string (third tab-split-selector))
```

Comment:
Why bother binding `gopher+string` if we're not going to do anything with it?

Reply:
That was one of the questions I had about this patch. I'll remove it then!